### PR TITLE
Cleaning up code to get to know the system

### PR DESCRIPTION
**Pull request by tilal6991**
Just some minor changes updating the SDK version to the newly released version and also cleaning up code to make it more readable.

### CODE REVIEW

File: library/src/main/java/com/mikepenz/materialdrawer/accountswitcher/AccountHeader.java [hunk 1]
```diff
@@ -31,9 +32,6 @@
 import java.util.Collections;
 import java.util.Stack;
 
-/**
- * Created by mikepenz on 27.02.15.
```
Comment (mikepenz) [inline library/src/main/java/com/mikepenz/materialdrawer/accountswitcher/AccountHeader.java]: please don't remove the created by comments :D

**Comment by mikepenz** [comment]
@tilal6991 thanks for the pull request. I'm currently preparing an update with a lot of changes and this pull request would conflict with those. sorry.

**Comment by tilal6991** [comment]
Sure not a problem. I have ideas for changing the internals of the system but let me know when you feel the system is stable for big PRs.

**Comment by mikepenz** [comment]
@tilal6991 currently building everything. after that some more testing. please first get in contact with me before you change the internals of the system ;)

**Comment by tilal6991** [comment]
Sure - where do you want to chat?

**Comment by mikepenz** [comment]
We can write some mails

Cheers,
Mike

**Pull request closed without merging**